apps/typegpu-docs/src/components/ControlPanel.tsx

```diff
@@ -167,7 +167,7 @@ export function ControlPanel() {
     <div
       className={cs(
         isGPUSupported ? '' : 'hidden md:flex',
-        'flex flex-col gap-4 p-6 bg-grayscale-0 rounded-xl max-h-[50%] md:max-h-full',
+        'box-border flex flex-col gap-4 p-6 bg-grayscale-0 rounded-xl max-h-[50%] md:max-h-full',
```

Comment:
could we instead of writing `box-border` and `m-0` everywhere add a global css reset?

something like:
```css
*, *::after, *::before {
  box-sizing: border-box;
  margin: 0;
}

or using tailwind \at apply

```

Reply:
We could, but I am wondering if then we would have to deal with components working differently between pages (since we cannot apply those resets to the Starlight docs page, due to those components relying on these defaults).

Reply:
I don't think we reuse any components between the docs and the landing/examples page, do we?

Reply:
Not yet 👀